install.bat

```diff
@@ -0,0 +1,77 @@
+@ECHO off
+
+ECHO "Welcome to the setup for mentorship-backend repo."
+
+ECHO "This installation requires Python version above 3.0. Make sure you have it installed."
+
+SET /P check_repo="Do you want to clone the repository [y/n] : "
+IF %check_repo%=="y" ( ECHO "Cloning the repository"
+git clone https://github.com/systers/mentorship-backend.git 
+) ELSE ( 
+ECHO "Make sure that you have the repository and the install.bat file in the same folder!" 
+)
+
+cd mentorship-backend
+
+ECHO "Installing the virtual environment - 'venv'"
+pip3 install virtualenv
+virtualenv venv
+
+CALL venv\Scripts\activate	
+
+ECHO "Activated virtual environment - 'venv'"
+
+ECHO "Installing the required modules"
+pip3 install -r requirements.txt
+IF %errorlevel% == 0 ( 
+goto :next 
+) ELSE ( 
+ECHO "Error in installing modules. Exited with status : %errorlevel%"
```

Comment:
I guess you need to put exit command here as the setup fails?

Reply:
I have transferred the control to the : steps function so that the user gets to know how to activate virtual env. Again if he wants to, the program would be exited from there automatically.

Reply:
Okay @Hardik0403! That would be fine too I guess. As @xprilion was reviwing your task, let him get back to the comments he left, once they are approved! I guess your task is complete. Great work :+1:

Reply:
Thank you for your feedback!
Yes sure, I would be waiting for the review.☺️

Reply:
In case it takes him more than expected time, some other mentor will approve the changes and finalize the task. Thanks for the patience. :smile:

Reply:
@Hardik0403 Also, you can submit it for the review on code-in now. As your part is done, next response is needed from mentor.

Reply:
No issues!! I remember he told that there is an Internet issue going on in his region, so I would be waiting.☺️